MFVNeutralino/plugins/VertexTracks.cc

```diff
@@ -934,14 +936,12 @@ bool MFVVertexTracks::filter(edm::Event& event, const edm::EventSetup& setup) {
     }
   }
   // now we get to do this all over for lepton tracks ... (with electrons and muons separate) 
-  /*
   if (use_separated_leptons) {
     for (size_t i = 0, im = all_muon_tracks->size(); i < im; ++i) {
       const reco::TrackRef& mtk = (*all_muon_tracks)[i];
-      //this will need to be changed once we have a different(?) track rescaler for leptons ... 
-      const auto rs = track_rescaler.scale(*mtk);
-      //copy/calculate the cheap things but now for muons ... 
-      const double p = mtk->p(); //don't need it yet 
+      const auto rs = (track_rescaler_which == 1) ? track_rescaler.scale(*mtk, "muon")
```

Comment:
@abbywarden
I guess this one only ever applies when using the lepton rescaling, since it's for muons specifically. But still, I would ask for equality with w_SingleLep instead of 1 (imagine if we ever changed the order of the enum!). We should also grep around to be sure that we haven't missed any instances of the rescaler that we initialize or check for equality using the index instead of the enum. It would be easy to make a mistake (and maybe use the wrong rescaling entirely) if we aren't careful.